api/controllers/FileUploadController.js

```diff
@@ -23,6 +23,12 @@ const FileUploadController = {
         const connectedToClamAV = await connectToClamAV(req);
         // @ts-ignore
         const userData = HelperService.getUserData(req, res);
+        const noFileUploadedError = Boolean(
+            req.flash('noFileUploadedError')[0]
+        );
+        const fileCountError = Boolean(req.flash('fileCountError')[0]);
```

Comment:
why not `req.flash('errors', ['my new error'])`, and every time there's a new error, you keep adding to `req.flash('error', [..])`? 

then, when you render, all you do is `errors: req.flash('errors')` 

also, Boolean check here is confusing. 

You can do `req.flash('errors', ['noFileUploadedError'])` as a string. Then, inside your template, you can do 
`if(errors.noFileUploadedError)`